client/signup/steps/domains/index.jsx

```diff
@@ -658,7 +658,7 @@ class DomainsStep extends Component {
 		const subHeaderPropertyName = 'signUpFlowDomainsStepSubheader';
 		const onboardingSubHeaderCopy =
 			siteType &&
-			[ 'onboarding', 'ecommerce-onboarding' ].includes( flowName ) &&
+			[ 'onboarding' ].includes( flowName ) &&
```

Comment:
Super nit for simpler code? 😄 

```suggestion
			flowName === 'onboarding' &&
```